suite-native/module-accounts-import/src/screens/AccountsImportScreen.tsx

```diff
@@ -44,15 +49,18 @@ export const AccountsImportScreen = ({
     AccountsImportStackRoutes.AccountImport,
     RootStackParamList
 >) => {
+    const { xpubAddress, currencySymbol } = route.params;
     const dispatch = useDispatch();
+    const deviceNetworkAccounts = useSelector(
+        selectDeviceNetworkAccounts(getMockedDeviceState(HIDDEN_DEVICE_ID), currencySymbol),
+    );
     const [accountInfo, setAccountInfo] = useState<AccountInfo | null>(null);
-    const [selectedDevice, setSelectedDevice] = useState<DummyDevice>();
-    const [assetName, setAssetName] = useState<string>('bitcoines #1');
+    const [accountName, setAccountName] = useState<string>(
```

Comment:
Use handler instead of `setAccountName`
- Input should be red when empty + disable confirm
- Max length on input (if exceeded -> red outline)

Reply:
Don't do that now, I am already started to prepare forms package and this should be handled as form, so after forms will be merged we will implement it here.